modules/multiplayer/multiplayer_spawner.h

```diff
@@ -91,7 +91,9 @@ class MultiplayerSpawner : public Node {
 	void _get_property_list(List<PropertyInfo> *p_list) const;
 #endif
 public:
-	Node *get_spawn_node() const { return spawn_node.is_valid() ? Object::cast_to<Node>(ObjectDB::get_instance(spawn_node)) : nullptr; }
+	Node *get_spawn_node() const {
+		return spawn_node.is_valid() ? Object::cast_to<Node>(ObjectDB::get_instance(spawn_node)) : nullptr;
+	}
```

Comment:
Took me way too long to track this down but this was actually caused by a bug in clang format. I kept this line though because it actually is cleaner. See on GitHub llvm/llvm-project# 54901 (I removed the link to not make unnecessary noise)